setup.cfg

```diff
@@ -10,6 +10,11 @@ long_description = file: README.rst
 long_description_content_type = text/x-rst
 edit_on_github = False
 github_project = astropy/specreduce
+install_requires = astropy, numpy, scipy, matplotlib, specutils
+# version should be PEP440 compatible (http://www.python.org/dev/peps/pep-0440)
+version = 0.0.dev
+minimum_python_version = 3.6
+
```

Comment:
it looks like this still needs to be changed to match APE17. specifically, `version` shouldn't be here and the stuff other than `astropy` are probably more appropriately put under `options.extra_requires`. i also strongly suggest we set the minimum python version to 3.7 so that we can use data classes where appropriate.

Reply:
Thanks, just checked out the doc at https://github.com/astropy/astropy-APEs/blob/master/APE17.rst and have updated accordingly.